src/AddressRegistry.Api.BackOffice/AddressController-Remove.cs

```diff
@@ -0,0 +1,104 @@
+    using Abstractions;
+    using Abstractions.Exceptions;
+    using Abstractions.Requests;
+    using Address;
+    using Be.Vlaanderen.Basisregisters.AggregateSource;
+    using Be.Vlaanderen.Basisregisters.Api.ETag;
+    using Be.Vlaanderen.Basisregisters.Api.Exceptions;
+    using FluentValidation;
+    using FluentValidation.Results;
+    using Infrastructure;
+    using Microsoft.AspNetCore.Http;
+    using Microsoft.AspNetCore.Mvc;
+    using StreetName;
+    using StreetName.Exceptions;
+    using Swashbuckle.AspNetCore.Filters;
+    using Validators;
+
+    public partial class AddressController
+    {
+        /// <summary>
+        /// Verwijder een adres.
+        /// </summary>
+        /// <param name="backOfficeContext"></param>
+        /// <param name="validator"></param>
+        /// <param name="ifMatchHeaderValidator"></param>
+        /// <param name="ifMatchHeaderValue"></param>
+        /// <param name="request"></param>
+        /// <param name="cancellationToken"></param>
+        /// <response code="202">Aanvraag tot verwijdering wordt reeds verwerkt.</response>
+        /// <response code="412">Als de If-Match header niet overeenkomt met de laatste ETag.</response>
+        /// <returns></returns>
+        [HttpPost("{persistentLocalId}/acties/verwijderen")]
+        [ProducesResponseType(StatusCodes.Status202Accepted)]
+        [ProducesResponseType(typeof(ProblemDetails), StatusCodes.Status404NotFound)]
+        [ProducesResponseType(typeof(ProblemDetails), StatusCodes.Status500InternalServerError)]
+        [ProducesResponseType(typeof(ProblemDetails), StatusCodes.Status500InternalServerError)]
+        [SwaggerResponseExample(StatusCodes.Status500InternalServerError, typeof(InternalServerErrorResponseExamples))]
+        public async Task<IActionResult> Remove(
+            [FromServices] BackOfficeContext backOfficeContext,
+            [FromServices] IValidator<AddressRemoveRequest> validator,
+            [FromServices] IIfMatchHeaderValidator ifMatchHeaderValidator,
+            [FromHeader(Name = "If-Match")] string? ifMatchHeaderValue,
+            [FromRoute] AddressRemoveRequest request,
+            CancellationToken cancellationToken = default)
+        {
+            await validator.ValidateAndThrowAsync(request, cancellationToken);
+
+            var addressPersistentLocalId =
+                new AddressPersistentLocalId(new PersistentLocalId(request.PersistentLocalId));
+
+            var relation = backOfficeContext.AddressPersistentIdStreetNamePersistentIds
+                .FirstOrDefault(x => x.AddressPersistentLocalId == addressPersistentLocalId);
+
+            if (relation is null)
+            {
+                return NotFound();
+            }
+
+            var streetNamePersistentLocalId = new StreetNamePersistentLocalId(relation.StreetNamePersistentLocalId);
+
+            try
+            {
+                // Check if user provided ETag is equal to the current Entity Tag
```

Comment:
Toggle check  to see if we go to SQS?

Reply:
Do we already want to add this is addres registry?